### DISCUSSION TITLE
Half-life not calculated

### DISCUSSION THREAD

**Discussion by jeslo328** [humanpred/pknca#314] [Q&A] [answered]
Hi

I was trying to calculate pk parameters from simulated data. 1 ID of the simulation (N=500) half life NA  [Too few points for half-life calculation (min.hl.points=3 with only 1 points)].
Is there a way to 're-calculate' the half life - or none and i only have to exclude this in stats calculation (N=499).
If there is a way to manipulate the half life - how can i do it.

Also can anyone direct me to any guidance on the stats of NCA. For eg. when half life cannot be calculated (lambda Z) - hence the AUCinf cannot be calculated. such when we are calculating the GM will we say the AUCinf as 0 OR as missing. 

Thank you

**Comment by billdenney** [comment, marked as answer]
The typical method would be to exclude the half life and other parameters that are NA from summary statistics.  With only one point for half life calculation, I can't think of a reasonable way to resolve it with NCA.  (You could simulate more points, but that wouldn't be reasonable in my opinion because the reason to perform NCA on simulation results is to mimic what you would have seen clinically, and it is atypical to have a ton of points measured in a clinical study.)

This exclusion is what is done when you run `summary()` on a calculated NCA result in PKNCA.
